### PR TITLE
added systemsdk/easy-log-bundle

### PR DESCRIPTION
**Pull request by dimadeush**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+easy_log:
+    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
+    # Use below if you're running application in container and want to log in stdout
+    #log_path: 'php://stdout'
+    max_line_length: 120
+    prefix_length: 2
+    ignored_routes: ['_wdt', '_profiler']
```
Comment (Nyholm) [inline systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml:7]: You dont need to show all your config. Just keep sensible defaults.

File: systemsdk/easy-log-bundle/1.10/config/packages/test/easy_log.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+easy_log:
```
Comment (Nyholm) [inline systemsdk/easy-log-bundle/1.10/config/packages/test/easy_log.yaml:1]: Same here

**Review by Nyholm — COMMENTED** [review]
Whenever a user install this package, they will get a big error message. (Like on travis)

Could we add the monolog handler config?

File: systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml (hunk 1 above)
Comment (dimadeush) [reply systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml:7]: It is already sensible defaults (we usually don't need to log debug info)

File: systemsdk/easy-log-bundle/1.10/config/packages/test/easy_log.yaml (hunk 2 above)
Comment (dimadeush) [reply systemsdk/easy-log-bundle/1.10/config/packages/test/easy_log.yaml:1]: It is already sensible defaults (we usually don't need to log debug info)

**Comment by dimadeush** [comment]
> Whenever a user install this package, they will get a big error message. (Like on travis)
> 
> Could we add the monolog handler config?

Yes, we can but there is some issue with it - Sometimes some projects can have edited default monolog config and it means that prepared monolog config here will not work as expected. 

So I think here I have two cases: 

1)delete checking monolog config in EasyLogExtension (row 40 https://github.com/systemsdk/easy-log-bundle/blob/master/src/DependencyInjection/EasyLogExtension.php) and that's exactly same error what you can see in travis CI here  

2)can try to add monolog config but in such case there can be some issues in case project have custom monolog config.

Do you have better ideas?

**Comment by dimadeush** [comment]
@fabpot @Nyholm  Selected case 1 mentioned above. Now CI passed, please review once again.

File: systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml (hunk 1 above)
Comment (Nyholm) [reply systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml:7]: I suggest you keep: 

```yaml
easy_log:
    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
    ignored_routes: ['_wdt', '_profiler']
 ```

The rest can be removed. There is no need to show all your config. 

Feel free to add a link to your documentation if you want to.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. I just want you to address my comments and I'll be happy to merge.

**Comment by dimadeush** [comment]
> Thank you. I just want you to address my comments and I'll be happy to merge.

Hello, done. Thank you in advance.

File: systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+easy_log:
+    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
+    # Use below if you're running application in container and want to log in stdout
+    #log_path: 'php://stdout'
```
Comment (Nyholm) [inline systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml:4]: I suggest you put this comment in your bundles documentation. The recipe is not a replacement for your documentation. 

Feel free to add a link to your docs.

**Review by Nyholm — CHANGES_REQUESTED** [review]

File: systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml (hunk 3 above)
Comment (dimadeush) [reply systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml]: Ok, done.

**Review by Nyholm — COMMENTED** [review]
Im happy with this. 
Just one more thing. Could you rename the files to `systemsdk_easy_log.yaml`?

**Comment by dimadeush** [comment]
> Im happy with this.
> Just one more thing. Could you rename the files to `systemsdk_easy_log.yaml`?

Done

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**